extensions/amp-story-auto-analytics/0.1/auto-analytics-configs.js

```diff
@@ -70,5 +70,35 @@ export const buildGtagConfig = (gtagId) => ({
         'repeat': false,
       },
     },
+    'storyShoppingPLPView': {
+      'on': 'story-shopping-plp-view',
+      'vars': {
+        'event_name': 'custom',
+        'event_action': 'story_shopping_plp_view',
+        'event_category': '${title}',
+        'event_label': 'page id: ${storyPageId}',
+        'send_to': [gtagId],
+      },
+    },
+    'storyShoppingPDPView': {
+      'on': 'story-shopping-pdp-view',
+      'vars': {
+        'event_name': 'custom',
+        'event_action': 'story_shopping_pdp_view',
+        'event_category': '${title}',
+        'event_label': 'product id: ${storyShoppingProductId}',
+        'send_to': [gtagId],
+      },
+    },
+    'storyShoppingBuyNowClick': {
+      'on': 'story-shopping-buy-now-click',
+      'vars': {
+        'event_name': 'custom',
+        'event_action': 'story_shopping_buy_now_click',
+        'event_category': '${title}',
+        'event_label': 'product id: ${storyShoppingProductId}',
+        'send_to': [gtagId],
+      },
+    },
```

Comment:
We need to add another event for shopping tag clicks. 
It could be called `storyShoppingTagClick`. It needs to send `storyShoppingProductId`.